app/src/main/java/de/danoeh/antennapod/activity/BugReportActivity.java

```diff
@@ -67,42 +72,60 @@ protected void onCreate(Bundle savedInstanceState) {
             clipboard.setPrimaryClip(clip);
             Snackbar.make(findViewById(android.R.id.content), R.string.copied_to_clipboard, Snackbar.LENGTH_SHORT).show();
         });
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.bug_report_options, menu);
+        return super.onCreateOptionsMenu(menu);
+    }
 
-        findViewById(R.id.btn_export_logcat).setOnClickListener(v -> {
-            try {
-                File filename = new File(UserPreferences.getDataFolder(null), "full-logs.txt");
-                filename.createNewFile();
-                String cmd = "logcat -d -f " + filename.getAbsolutePath();
-                Runtime.getRuntime().exec(cmd);
-                //share file
+    @Override
+    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
+        if (item.getItemId() == R.id.export_logcat) {
+            AlertDialog.Builder alertBuilder = new AlertDialog.Builder(this);
+            alertBuilder.setTitle("Warning!").setMessage("Logs may contain sensitive information");
```

Comment:
Please make this two lines to be more consistent.

```suggestion
            alertBuilder.setTitle("Warning!");
            alertBuilder.setMessage("Logs may contain sensitive information");
```